fern/api-reference/hyperevm/hyperevm-api-quickstart.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Hyperevm API Quickstart
+description: How to get started building on Hyperevm and using the JSON-RPC API
+subtitle: How to get started building on Hyperevm and using the JSON-RPC API
+url: https://docs.alchemy.com/reference/hyperevm-api-quickstart
+slug: reference/hyperevm-api-quickstart
+---
+
+*To use the Hyperevm API you'll need to [create a free Alchemy account](https://dashboard.alchemy.com/) first!*
+
+## Introduction
+
+HyperEVM is a high-performance, Ethereum-compatible execution layer developed by HyperLiquid that enables smart contracts to run natively alongside its decentralized trading engine, combining low-latency performance with full EVM compatibility.
+
+## What is the Hyperevm API?
+
+The Hyperevm API allows interaction with the Hyperevm network through a set of JSON-RPC methods. Its design is familiar to developers who have worked with Ethereum's JSON-RPC APIs, making it intuitive and straightforward to use.
+
+## Getting Started Instructions
+
+### 1. Choose a Package Manager (npm or yarn)
+
+Select a package manager to manage your project's dependencies. Choose between `npm` and `yarn` based on your preference or project requirements.
+
+<CodeGroup>
+  ```shell npm
+  # Begin with npm by following the npm documentation
+  # https://docs.npmjs.com/downloading-and-installing-node-js-and-npm
+  ```
+
+  ```shell yarn
+  # For yarn, refer to yarn's installation guide
+  # https://classic.yarnpkg.com/lang/en/docs/install
+  ```
+</CodeGroup>
+
+### 2. Set Up Your Project
+
+Open your terminal and execute the following commands to create and initialize your project:
+
+<CodeGroup>
+  ```shell npm
+  mkdir hyperevm-api-quickstart
+  cd hyperevm-api-quickstart
+  npm init --yes
+  ```
+
+  ```shell yarn
+  mkdir hyperevm-api-quickstart
+  cd hyperevm-api-quickstart
+  yarn init --yes
+  ```
+</CodeGroup>
+
+This creates a new directory named `hyperevm-api-quickstart` and initializes a Node.js project within it.
+
+### 3. Make Your First Request
+
+Install Axios, a popular HTTP client, to make API requests:
+
+<CodeGroup>
+  ```shell npm
+  npm install axios
+  ```
+
+  ```shell yarn
+  yarn add axios
+  ```
+</CodeGroup>
+
+Create an `index.js` file in your project directory and paste the following code:
+
+<CodeGroup>
+  ```javascript index.js
+  const axios = require('axios');
+
+  const url = 'https://hyperliquid-mainnet.g.alchemy.com/v2/${your-api-key}';
```

Comment:
seems like it should be backticks instead of single quote

Reply:
this was requested by node services to not be backtick